packages/checkout/sdk/src/smartCheckout/sell/sell.ts

```diff
@@ -130,18 +131,19 @@ export const sell = async (
     orderbook = instance.createOrderbookInstance(config);
     const { seaportContractAddress } = orderbook.config();
     spenderAddress = seaportContractAddress;
-    const sellItem = sellToken.type === ItemType.ERC721
+
+    const sellItem = 'type' in sellToken && sellToken.type === ItemType.ERC1155
       ? {
         type: sellToken.type,
```

Comment:
given there are 2 distinctive types now, this benefits from explicitly setting the type to `ItemType.ERC1155`

Reply:
done!